src/main/java/com/cleanroommc/groovyscript/sandbox/GroovyScriptSandbox.java

```diff
@@ -61,10 +65,10 @@ public GroovyScriptSandbox(File scriptRoot, File cacheRoot) throws MalformedURLE
         registerBinding("Mods", ModSupport.INSTANCE);
         registerBinding("Log", GroovyLog.get());
         registerBinding("EventManager", GroovyEventManager.INSTANCE);
+        registerBinding("MinecraftHome", GroovyScript.getMinecraftHome());
```

Comment:
given its already in `GroovyHelpers as `getMinecraftHome()`, dont think this is needed